### DISCUSSION TITLE
Feature request: Diff viewed/remaining meter UI

### DISCUSSION THREAD

**Discussion by jkillian** [community/community#11251] [Pull Requests]
When marking files as viewed, a helpful meter shows you how many files you have left to review:

<img width="432" alt="image" src="https://user-images.githubusercontent.com/1400151/153233736-66b8371e-88a5-4759-977b-b8840d8b16d8.png">

However, the number of files left is not an amazing metric - you may have reviewed the one file with 95% of the changes and be mostly done, but the current UI will make it look like you still have most of the PR left to review.

I think it'd be great if the same UI also showed you the lines of diff left to review. Something perhaps like "+650 -230 left to review" with the normal green/red coloring scheme.

It's just a small tweak, but I think it would help provide a sense of motivation and progress while going through code reviews of very large PRs!

**Comment by ValentinH** [comment]
I was also thinking about this today, it would be a great addition!

**Comment by toniopelo** [comment]
That would definitely help, I'm all for it :)
